File: check-tables.py
#!/usr/bin/env python3
import pg8000.dbapi

conn = pg8000.dbapi.connect(
    host='episode-control-dev.csnow208wqtv.us-east-1.rds.amazonaws.com',
    user='postgres',
    password='EpisodeControl2024!Dev',
    database='episode_metadata',
    port=5432
)

cursor = conn.cursor()
cursor.execute("SELECT table_name FROM information_schema.tables WHERE table_schema = 'public' ORDER BY table_name")
tables = cursor.fetchall()

print("\nTables in episode_metadata database:")
if tables:
    for table in tables:
        print(f"  - {table[0]}")
else:
    print("  No tables found")

cursor.close()
conn.close()

File: check-tables.js
const pg = require('pg');

const pool = new pg.Pool({
  host: '127.0.0.1',
  user: 'postgres',
  password: 'Ayanna123',
  database: 'episode_metadata',
});

pool.query(`
  SELECT column_name, data_type
  FROM information_schema.columns
  WHERE table_name = 'episode_wardrobe_defaults'
  ORDER BY ordinal_position
`, (err, res) => {
  if (err) {
    console.error('Error:', err);
  } else {
    console.log('Columns in episode_wardrobe_defaults:');
    res.rows.forEach(col => {
      console.log(`  - ${col.column_name}: ${col.data_type}`);
    });
    
    const hasDeletedAt = res.rows.some(col => col.column_name === 'deleted_at');
    console.log(`\n${hasDeletedAt ? '✅' : '❌'} deleted_at column ${hasDeletedAt ? 'EXISTS' : 'MISSING'}`);
  }
  pool.end();
});



